### ISSUE TITLE
Add staking info to rc

### ISSUE THREAD

**Issue by majestrate** [oxen-io/lokinet#1960]
Some additional hints should be added to the lokinet rc including but not limited to.

- Hash of operator wallet
- hashes of contributing stake wallets (maybe include portion?)
- Timestamp of expected stake unlock when applicable

**Comment by majestrate** [comment]
in #2168 we will try another way to do this, mainly by adding more node to node messages that can help with this.

**Comment by jagerman** [comment]
I don't think this should be closed until #2168 is finalized with these capabilities.
